charts/spire/charts/spire-server/values.yaml

```diff
@@ -206,23 +204,22 @@ nodeAttestor:
 
 # tornjak - Tornjak specific configuration
 tornjak:
+  # -- enable, to deploy Tornjak API (backend) component
   enabled: false
-  # image - Tornjak image (frontend + backend) if not separated above
-  image: # ghcr.io/spiffe/tornjak
+  # -- Tornjak API image
+  image:
     registry: ghcr.io
-    repository: spiffe/tornjak
+    repository: spiffe/tornjak-be
     pullPolicy: IfNotPresent
-    # Overrides the image tag whose default is the chart appVersion.
-    # TODO we should use a specific Tornjak version instead of 'latest'
-    version: "latest"
+    # -- Overrides the image tag whose default is the chart appVersion.
+    version: "v1.0.1"
   config:
-    # Front-end specific configuration:
-    frontend:
-      # apiServerURL - URL of the Tornjak back-end
-      apiServerURL: "http://localhost:10000"  # 👈 Use it for minikube or kind
-     # Back-end specific configuration
+    # Tornjak Backend specific configuration
     backend:
-      # dataStore - persistent DB for storing Tornjak specific information
+      # -- URL of the Tornjak APIs (backend).
+      # The default value is used when both deployed in the same, local cluster.
+      apiServerURL: "http://localhost:10000/"  # 👈 Use it for minikube or kind
```

Comment:
Probably want to do an ingress that defaults to the same domain as we use for other defaults (`example.org`) e.g. `tornjak.example.org` so it is aligned with the rest of the chart.

Reply:
tornjak-api.example.org perhaps.

Reply:
We display it to the user at the end of the NOTES.txt so they can either use it directly (if Ingress defined) or they can set up a port-forward